src/common/config/state/runtime.js

```diff
@@ -4,12 +4,13 @@ import { gosNREUMInitializedAgents } from '../../window/nreum'
 import { getCurrentSessionIdOrMakeNew } from '../../window/session-storage'
 import { getConfigurationValue } from '../config'
 import { globalScope } from '../../util/global-scope'
-import { BUILD_ENV, VERSION } from '../../constants/environment-variables'
+import { BUILD_ENV, DIST_METHOD, VERSION } from '../../constants/environment-variables'
```

Comment:
Since these are set at compile time.. do they belong in the `runtime` object, or does `info` make more sense?

Reply:
We should probably work to standardize the configs in the future, but imo 

info should be user configs related to account and agent data

init should be configurations of the agent and features 

runtime should be a place where state is kept and shared 

Definitely open to other ideas

Reply:
By that, it sounds like these should go in `init` then (`runtime` in my head are dynamic internal fields determined at run time--nothing statically known should be in there), but not important atm so resolving this.